dask/array/overlap.py

```diff
@@ -3,7 +3,7 @@
 from operator import getitem
 from itertools import product
 from numbers import Integral
-
+from numpy import isscalar
```

Comment:
Could we instead use `import numpy as np` here and then call `np.isscalar`. This will be consistent with our current numpy import usage throughout the rest of the codebase.

Reply:
Agreed. Though we might be able to drop this entirely with my suggestion below.

Reply:
Integral works